src/oom/test-oomd-util.c

```diff
@@ -291,19 +291,19 @@ static void test_oomd_pressure_above(void) {
 
 static void test_oomd_mem_and_swap_free_below(void) {
         OomdSystemContext ctx = (OomdSystemContext) {
-                .mem_total = 20971512 * 1024U,
-                .mem_used = 3310136 * 1024U,
-                .swap_total = 20971512 * 1024U,
-                .swap_used = 20971440 * 1024U,
+                .mem_total = 20971512ULL * 1024U,
```

Comment:
hmm, please use UINT64_C() for this, as that's what the used types here want, they are uint64_t